mobile/src/stateMachine/views/main/home.tsx

```diff
@@ -239,6 +271,7 @@ export class HomeView extends ViewState<{ opacity: Animated.Value }> {
                                 onPress={() => this.onTipItemPress(s)}
                             />
                         ))}
+                        {/* add logic to render certain things in check in spot depending on state of partial save*/}
```

Comment:
If this is a todo, please prefix with "TODO: "

Reply:
This specific todo is covered in PR #79 so will be removed here

Reply:
removed